hr_period/models/hr_contract.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding:utf-8 -*-
+# Copyright 2015 Savoir-faire Linux. All Rights Reserved.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+from .hr_fiscal_year import get_schedules
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    # Add semi-monthly to payroll schedules
+    shedule_pay = fields.Selection(
```

Comment:
rename the field (s/shedule/schedule/). If the module exists in older versions of odoo, use oldname to ease the migration.